### PR TITLE
[improvement](insert) Modify insert into table_id(xxx) to doris_inter…

### PR DESCRIPTION
**Pull request by mymeiyi**
…nal_table_id(xxx)

## Proposed changes

Modify keyword `table_id` to a special keyword `doris_internal_table_id`

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by mymeiyi** [comment]
run buildall

**Comment by zzzzzzzs** [comment]
LGTM

**Review by zzzzzzzs — APPROVED** [review]

**Review by dataroaring — DISMISSED** [review]
LGTM

**Comment by mymeiyi** [comment]
run buildall

File: fe/fe-core/src/main/cup/sql_parser.cup [hunk 1]
```diff
@@ -347,6 +347,7 @@ terminal String
     KW_DISTRIBUTION,
     KW_DIV,
     KW_DO,
+    KW_DORIS_INTERNAL_TABLE_ID,
```
Comment (morrySnow) [inline fe/fe-core/src/main/cup/sql_parser.cup:350]: should add into non-reserved keyword list, and add same keyword for Nereids in `src/main/antlr4/org/apache/doris/nereids/DorisParser.g4`

**Comment by mymeiyi** [comment]
run buildall

**Comment by mymeiyi** [comment]
run buildall

**Comment by mymeiyi** [comment]
run p0

**Review by dataroaring — APPROVED** [review]
LGTM

**Pull request merged**